libdirections-common/src/main/java/com/mapbox/navigation/route/common/NavigationRouteCallback.kt

```diff
@@ -30,7 +31,8 @@ internal class NavigationRouteCallback(
         return response.body()?.routes()?.isNotEmpty() ?: false
     }
 
+    // TODO move NavigationTelemetry to separate module
```

Comment:
This class should that the metrics reporter interface as parameter.

Reply:
discussed, postponed to separate PR